runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -2066,6 +2262,32 @@ void J9::ARM64::PrivateLinkage::buildVirtualDispatch(TR::Node *callNode,
          ListIterator<J9::ARM64PICItem>  i(&values);
          J9::ARM64PICItem               *pic = i.getFirst();
 
+         if (methodSymbol->isInterface())
+            {
+            // Find the class pointer to the interface class if it is already loaded.
+            //
+            TR::Method *interfaceMethod = methodSymbol->getMethod();
+            int32_t len = interfaceMethod->classNameLength();
+            char *s = TR::Compiler->cls.classNameToSignature(interfaceMethod->classNameChars(), len, comp());
+            auto interfaceClassOfMethod = fej9->getClassFromSignature(s, len, methodSymRef->getOwningMethod(comp()));
+            int32_t numStaticPICSlots = (pic->_frequency > MAX_PROFILED_CALL_FREQUENCY) ? 1 : values.getSize();
+
+            // Disable lastITable logic if all the implementers can fit into the pic slots during non-startup state
+            if (useLastITableCache && interfaceClassOfMethod && comp()->getPersistentInfo()->getJitState() != STARTUP_STATE)
```

Comment:
How about moving the condition `useLastITableCache` to the outer `if` statement?
```
if (methodSymbol->isInterface() && useLastITableCache)
```

Reply:
I moved `useLastITableCache` to the outer `if` statement.